sherpa/models/model.py

```diff
@@ -1076,56 +1092,47 @@ def html_model(mdl):
         out += '<th>{}</th>'.format(col)
 
     out += '</tr></thead><tbody>'
-    seen = set([])
-    mcount = 1
-    for par in mdl.pars:
-        if par.hidden:
-            continue
-
-        def addtd(val):
-            "Use the parameter to convert to HTML"
-            return '<td>{}</td>'.format(par._val_to_html(val))
-
-        mname = par.modelname
-
-        style = ''
-        if mname not in seen:
-            if len(seen) > 0:
-                style = ' class="block"'
-
-        out += '<tr{}>'.format(style)
-        if mname not in seen:
-            cls = "model-" + ("even" if mcount % 2 == 0 else "odd")
-            out += '<th class="{}" scope="rowgroup" '.format(cls)
-            out += 'rowspan={}>{}</th>'.format(nrows[mname], mname)
-            seen.add(mname)
-            mcount += 1
-
-        out += '<td>{}</td>'.format(par.name)
-
-        linked = par.link is not None
-        if linked:
-            out += "<td>linked</td>"
-        else:
-            out += '<td><input disabled type="checkbox"'
-            if not par.frozen:
-                out += ' checked'
-            out += '></input></td>'
 
-        out += addtd(par.val)
+    for mcount, n, comp in zip(range(len(complist)), nrows, complist):
```

Comment:
Would (the untested)

    for mcount, (n, comp) in enumerate(zip(nrows, complist)):

work / be better?

Reply:
it looks cleaner!